mobile/src/controllers/LocalSettings.ts

```diff
@@ -131,6 +152,39 @@ export class LocalSettingsController implements ILocalSettingsController {
             }
         });
     }
+
+    updateQolOnboarding(diff: Partial<QolSettings>) {
+        const qol = this.current.qol || { };
+        transaction(() => {
+            let changed = transferChangedFields(diff, qol, "seenOnboardingQol", 'lastFullQol');
```

Comment:
@MarkDavies0 thinking about the discussion before, should this instead be "seenQolOnboarding" instead of "seenOnboardingQol" because we are setting the fact that they have seen the onboarding for qol, not the qol itself.

Reply:
Yes that is the most clear